seller-app-api/services/v2/product.service.js

```diff
@@ -80,11 +80,11 @@ class ProductService {
 
             if (!category) {
                 category = {
-                    "name": "Grocery",
-                    "domain": "ONDC:RET10"
+                    "name": "F&B",
+                    "domain": "ONDC:RET11"
```

Comment:
If you dont get any category dont generate on_search

Reply:
ok returning false  here